src/main/kotlin/no/fdk/concept_catalog/service/ConceptMappers.kt

```diff
@@ -67,6 +67,7 @@ fun BegrepDBO.addUpdatableFieldsFromDTO(dto: Begrep) =
     copy(
         status = dto.status,
         statusURI = dto.statusURI,
+        versjonsnr = dto.versjonsnr ?: versjonsnr,
```

Comment:
Trenger ikke elvis her, siden en får bad request om versjonsnr ikke har blitt sendt inn
```suggestion
        versjonsnr = dto.versjonsnr,
```

Reply:
dto.versjonsnr er optional

Reply:
aha, ja. Alt der er jo type optional, det føles veldig unødvendig. Men sikkert en større jobb å gjøre noe med det. Ignorer maset mitt her og kjør :+1: 

Vi kan vurdere å gjøre dto mer strict senere en gang